### ISSUE TITLE
ConvertTo-PnPClientSidePage - cannot bind positional elements because no names were given

### ISSUE THREAD

**Issue by webbrewer** [SharePoint/sp-dev-modernization#402]
Thank you for reporting an issue or suggesting an enhancement. We appreciate your feedback - to help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you. Thanks!

#### Category

[x ] Bug
[ ] Enhancement

#### Problem Area

[ ] Page Transformation: Error during the setup/use of the Page Transformation UI solution (did you check our [troubleshooting guide](https://github.com/SharePoint/sp-dev-modernization/blob/dev/Solutions/PageTransformationUI/docs/troubleshootingguide.md)?)
[x ] Page Transformation: Error during the use of page transformation from PnP PowerShell
[ ] Page Transformation: Error during the use of page transformation from .Net
[ ] Page Transformation: Page is not looking correct after transformation
[ ] Modernization Scanner: something went wrong...

#### Expected or Desired Behavior

Convert classic publishing sub site blog post page to news page in modern communications site

#### Observed Behavior

PS error: cannot bind positional elements because no names were given

#### Steps to Reproduce

Add postcategory column to sitepages library in comms site
Run Connect-PnPOnline -Url https://xxx.sharepoint.com/classic blog site 
Run ConvertTo-PnPClientSidePage -BlogPage -Identity "2019 Reflections" -Overwrite -TargetWebUrl https://xxx.sharepoint.com/sites/new comms site -LogType File -LogVerbose -LogFolder "c:\temp" -KeepPageCreationModificationInformation -PostAsNews -SetAuthorInPageHeader -CopyPageMetadata
#### Submission Guidelines

  - The output of the following 2 PnP PowerShell statements: `Get-PnPStorageEntity` 
No output
and `Get-PnPTenantServicePrincipalPermissionGrants`
error value cannot be null
get-module -Name SharePointPnPPowerShellOnline Binary version 3.13.19 (I installed latest version but still shows older one)

Thanks for your contribution! Sharing is caring.

**Comment by jansenbe** [comment]
Hi @webbrewer ,

How did you define the PostCategory field on the target site? We do support either a Choice field or a text field, so no LookUp column as it was on the target. See https://docs.microsoft.com/en-us/sharepoint/dev/transform/modernize-blogs#step-22-configuring-the-communication-site for some more context. Also ensure that in the case of a choice field all the blog categories are correctly added.

If that does not help then please provide me a log output of the transformation run: your command line shows you're already creating logs, it would help if you share the created MD file

**Comment by webbrewers1** [comment]
This is webbrewers - had to sign in with my work account....
I created the category column as a choice field initially without categories specified as that wasn't mentioned in the tutorial, but even with them added, still get the same error.
There are no log files listed under c:\temp
FYI The classic site is a sub site of root site using a custom master page. I attached the full PS error.
![Untitled](https://user-images.githubusercontent.com/54376858/72006063-dad3ad00-321c-11ea-82b4-5ca0e1035c03.png)

**Comment by webbrewers1** [comment]
I found out that even though I'd installed the latest SP pnp powershell online version, it still showed 3.10.1906.0. I ran 
Update-Module SharePointPnPPowerShell* 
and it installed it again and when checked now showed the old and new version 3.16.1912.0. When I checked again, both versions were the latest.
After that running ConvertTo-PnPClientSidePage again did succeed in transforming a blog page. 
So in my case at least the fix was to make sure the latest ps versions were installed which doesn't seem to happen as you might expect.
